contracts/modules/cover/Cover.sol

```diff
@@ -637,12 +637,14 @@ contract Cover is ICover, MasterAwareV2, IStakingPoolBeacon, ReentrancyGuard {
     }
   }
 
-  function editProductsIpfsMetadata(
+  function editProducts(
     uint[] calldata productIds,
+    Product[] calldata newProducts,
     string[] calldata ipfsMetadata
   ) external override onlyAdvisoryBoard {
 
     for (uint i = 0; i < productIds.length; i++) {
+      _products[productIds[i]] = newProducts[i];
```

Comment:
The `Product` struct contains the following fields:
- `uint16 productType`
- `address productAddress`
- `uint32 coverAssets`
- `uint16 initialPriceRatio`
- `uint16 capacityReductionRatio`

As a result:
1. I don't find it OK to allow to change `productType` and `productAsset`.
2. There's no validation whatsoever on the correctness of the other parameters.